documentation/ssl.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: documentation
+menu: ssl
+subtitle: SSL support
+---
+# SSL support
+
+In many scenarios, maintaining a secure connection to the database is important. Flyway can easily be configured to
+use SSL to establish a secure connection as and when required, provided the relevant database and JDBC driver also 
+support SSL. Although details will vary between databases, the process to configure them is as follows:
+
+## Obtain a copy of the database's certificate 
+
+For on-premise databases, the certificate that you need to establish a trusted connection will have been installed 
+with the database and should be obtained from the relevant administrator. For cloud services, the provider will 
+publish the relevant certificate - for example [Azure](https://www.digicert.com/CACerts/BaltimoreCyberTrustRoot.crt.pem)
+and [Amazon RDS](https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem).
+
+## Import the certificate into a truststore
+
+Your certificate is very likely already in a form in which it can be imported into a local trust store 
+(for example, `.pem` or `.der`). If not, then it can be converted using a tool such as `openssl`.
+
+To import the certificate, use the `keytool` utility. This is included as part of the Java runtime environment that 
+is shipped with Flyway. If you're making a new store then you will be prompted for a password. Don't lose it - you
+will need that password later!
+
+<pre class="console"><span>&gt;</span> keytool -keystore myStorePath -alias "My database certificate" -import -file databaseCertificate.pem</pre>
+
+If you don't specify a particular truststore then the default location is in your home directory: 
+`$HOME/.keystore` (Linux) or `%USER_HOME%\.keystore` (Windows)
+
+You can also check which certificates have been imported already:
+
+<pre class="console"><span>&gt;</span> keytool -keystore myStorePath -list</pre>
+
+## Make sure Java can access the truststore
+
+You now need to configure your Java environment to be able to access these stores. If you're using the Flyway 
+command-line, then you can simply set the `JAVA_ARGS` environment variable which the command-line script reads.
+
+<pre class="console"><span>&gt;</span> JAVA_ARGS='-Djavax.net.ssl.trustStore="myStorePath" -Djavax.net.ssl.trustStorePassword="myStorePassword"'</pre>
```

Comment:
I think it makes sense to link to the blog post about `JAVA_ARGS` https://flywaydb.org/blog/jvm-properties